### PR TITLE
Updated the README.md file with additional hints

### PR DESCRIPTION
**Pull request by mircorudolph**
I updated the README.md file with additional hints based on my experience when setting up the local dev environment on my system.

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -96,6 +123,10 @@ npm run create-admin
 npm run dev
 ```
 
+The standard port is 3000 and the application can be opend at http://localhost:3000. Please check the log outputs, if a different port has been used.
+
+Use `johndeo@example.com` and `password` to login.
```
Comment (lemilonkh) [inline app/README.md:128]: I don't think we create this example account? You need to sign up first.

Comment (lemilonkh) [reply app/README.md:128]: Also it's johndoe

**Review by lemilonkh — APPROVED** [review]
Thanks 🙌🏼✨

**Review by cephaschapa — APPROVED** [review]

File: app/README.md (hunk 1 above)
Comment (mircorudolph) [reply app/README.md:128]: In the example.env file it is listed as:

DEFAULT_ADMIN_EMAIL=johndeo@example.com
DEFAULT_ADMIN_PASSWORD=password

For me it worked like this

Comment (mircorudolph) [reply app/README.md:128]: That is happening in the script 

```
## set up login in credentials
npm run create-admin
```

**Pull request merged**